pep-0001.txt

```diff
@@ -696,7 +696,7 @@ For each new PEP that comes in an editor does the following:
 
 * The file name extension is correct (i.e. ``.rst``).
 
-* Make sure the appropriate core developer(s) is added to
+* Make sure the appropriate sponsor or co-author(s) are added to
```

Comment:
```suggestion
* Make sure the appropriate sponsor and co-author(s) are added to
```

Per the above text, both should be added, not one or the other as "or" might imply

Reply:
Here I disagree (from logic this is ∨ not ⊻) -- we don't say "either sponsor or co-author(s)", and there are many PEPs without sponsors (core dev author). "or" in this context is therefore one of P, Q, P+Q.

Reply:
Huh...I (AmE) at least read this as an exclusive or, i.e. implying that either P, Q but not P + Q, whereas with "and" I read it as the union of the sets P and Q intersected with the set of users with write access to `python/peps` (even if set P may be empty and the result is logically equivalent to only Q). But if others read it differently and "or" reads clearer for the intended meaning, then I guess I'm the odd CAM out, heh.